matrix_client/api.py

```diff
@@ -179,6 +188,10 @@ def create_room(self, alias=None, is_public=False, invitees=None):
             content["room_alias_name"] = alias
         if invitees:
             content["invite"] = invitees
+        if name:
```

Comment:
Should we be adding even more unspecced fields to this? I'm fine with it if these are actually gonna be in the spec, but haven't been documented yet, but I want to make sure we aren't writing to synapse bugs. @Half-Shot, do you know what the status of these fields are?